app/tests/src/transaction-planner.js

```diff
@@ -106,7 +108,7 @@ describe('Succesful transaction plans', () => {
     it(`should ${name}`, () => {
       const {utxos, changeAddress, fee, args} = setting
       const txPlanResult = selectMinimalTxPlan(utxos, changeAddress, args)
-      if (txPlanResult.success === true) {
+      if (isTxPlanResultSuccess(txPlanResult)) {
```

Comment:
I would rather replace the one-line helper with its explicit implementation as it doesn't really simplify the logic, but it arguably adds one more layer of indirection to whoever wants to understand what this check does. Also it seems a bit weird for a function that seems t expect a tx plan to also allow null/undefined and hide it in its signature, possibly misleading someone casually checking this code that the tx plan must be defined